src/libs/Navigation/NavigationRoot.tsx

```diff
@@ -109,22 +115,17 @@ function NavigationRoot(props) {
         }
 
         statusBarAnimation.value = 0;
-        statusBarAnimation.value = withDelay(
-            300,
-            withTiming(1, {
-                duration: 300,
-                easing: Easing.in,
-            }),
-        );
+        statusBarAnimation.value = withDelay(300, withTiming(1));
```

Comment:
Why did you get rid of the other `withTiming` configs?

Reply:
Answered [here](https://github.com/Expensify/App/pull/31543#discussion_r1402174160) 😄